app/features/feedback/shared/strategies/radial/create-rule.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+function createRule(subjectRule, workflowRule) {
+  const rule = {
+    failureEnabled: workflowRule.failureEnabled,
+    id: subjectRule.id,
+    strategy: workflowRule.strategy,
+    successEnabled: workflowRule.successEnabled,
+    tolerance: subjectRule.tolerance || workflowRule.defaultTolerance,
+    x: subjectRule.x,
+    y: subjectRule.y
+  };
+
+  if (rule.failureEnabled) {
+    rule.failureMessage = subjectRule.failureMessage ||
+      workflowRule.defaultFailureMessage;
+  }
+
+  if (rule.successEnabled) {
+    rule.successMessage = subjectRule.successMessage ||
+      workflowRule.defaultSuccessMessage;
+  }
+
+  if (_.every(rule, property => property)) {
```

Comment:
not sure if this is the cause of 2, but I see the following from this line which seems strange:
![screen shot 2017-12-04 at 4 24 16 pm](https://user-images.githubusercontent.com/12118751/33579546-d99c80c8-d90f-11e7-994f-e8bb6dcb6d1d.png)